.github/workflows/deploy.yml

```
@@ -0,0 +1,36 @@
+name: deploy
+
+on:
+  push:
+    tags:
+      - "[0-9]+.[0-9]+.[0-9]+"
+
+jobs:
+
+  deploy:
+    if: github.repository == 'pytest-dev/pytest-qt'
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+        persist-credentials: false
+
+    - name: Set up Python
+      uses: actions/setup-python@v3
+      with:
+        python-version: "3.10"
```

```suggestion
        python-version: "3.11"
```

now that it was released maybe?